docs/modelserving/v1beta1/rollout/canary-example.md

```diff
@@ -0,0 +1,256 @@
+# Canary Rollout Example
+
+## Setup
+
+1. Your ~/.kube/config should point to a cluster with [KServe installed](https://github.com/kserve/kserve#installation).
```

Comment:
let's point to the serverless installation guide as the canary feature is only supported there.
https://kserve.github.io/website/master/admin/serverless/

Reply:
changed link